lib/mm_network.pm

```diff
@@ -172,6 +173,10 @@ sub check_ip_in_subnet {
 
 sub setup_static_mm_network {
     my $ip = shift;
+    if (is_opensuse) {
```

Comment:
Can you run some extra tests for nfs. As this function is used by yast2_nfs4_client and some other modules as well

Reply:
The other modules using it, appart from nfs, only run on SLE, so no problem. But indeed, the test will fail for nfs on TW, as it already has wicked by default. Thanks, I'm adding check_var('DESKTOP', 'textmode').